packages/cli/src/tools/packageManager.ts

```diff
@@ -18,7 +17,7 @@ const packageManagers = {
     install: ['add'],
     installDev: ['add', '-D'],
     uninstall: ['remove'],
-    installAll: ['install'],
+    installAll: ['install', '--no-immutable'],
```

Comment:
I know that this line shouldn't be here but I was just testing if CI will install packages when the `--no-immutable` flag, and it will. (Right now it is failing randomly because of some flakiness regarding Android tests). Do you have maybe some ideas how we should workarounds this, because we can't simply pass `--skip-install` when initialising project in CI, because when creating project even with `--skip-install` we install packages in `/tmp` directory get the template from `node_modules`. We need to figure out to somehow pass this flag when installing packages while creating project.

Reply:
Does `YARN_ENABLE_IMMUTABLE_INSTALLS=false` work?

![image](https://github.com/react-native-community/cli/assets/4123478/e362a39f-e27c-47e8-9fc1-94c38975818b)

Reply:
Ah, I missed this. Thanks Tommy! 🙌